orchagent/pfcwdorch.cpp

```diff
@@ -477,9 +479,10 @@ bool PfcWdSwOrch<DropHandler, ForwardHandler>::registerInWdDb(const Port& port,
 {
```

Comment:
We need some comments in here since now 'losslessTc' var will contain all priorities in the case of asym pfc

Reply:
Added below where ```losslessTc``` is declared.